nixos/modules/config/malloc.nix

```diff
@@ -10,7 +10,15 @@ let
       libPath = "${pkgs.graphene-hardened-malloc}/lib/libhardened_malloc.so";
       description = ''
         An allocator designed to mitigate memory corruption attacks, such as
-        those caused by use-after-free bugs.
+        those caused by use-after-free bugs - Default variant.
+      '';
+    };
+
+    graphene-hardened-light = {
```

Comment:
From just reading the options I do not know what the difference is here

Reply:
I'll try to rewrite both of these then